managed/src/main/java/com/yugabyte/yw/scheduler/Scheduler.java

```diff
@@ -251,6 +254,17 @@ private void runMultiTableBackupsTask(Schedule schedule) {
   }
 
   private void runDeleteBackupTask(Customer customer, Backup backup) {
+    UUID universeUUID = backup.getBackupInfo().universeUUID;
+    try {
+      Universe universe = Universe.get(universeUUID);
+      if (universe.getUniverseDetails().universePaused) {
```

Comment:
I was expecting this filtering in `getExpiredBackups` 
That's where rest of the filtering is done.